createDerby.groovy

```diff
@@ -430,6 +435,37 @@ new File("cs2wikidata.csv").eachLine { line,number ->
 }
 unitReport << "  <testcase classname=\"WikidataCreation\" name=\"ChemSpiderFound\"/>\n"
 
+//// IUPHAR registry numbers
```

Comment:
This is the only place where we should have iupharDS:

Reply:
So only reading in from Wikidata, got it.